Package.swift

```diff
@@ -68,6 +73,14 @@ let package = Package(
                 "Datadog",
                 .product(name: "CrashReporter", package: "PLCrashReporter"),
             ]
-        )
+        ),
+        .target(
+            name: "DatadogAlamofireExtension",
+            dependencies: [
+                "Datadog",
+                "Alamofire"
+            ],
+            path: "Sources/DatadogExtensions/Alamofire/"
```

Comment:
We need to specify the `path` because this code does not live at the default location where Swift Package Manager would look for this target: `Sources/DatadogAlamofireExtension`.